packages/browser-wallet/src/popup/pages/Account/ConfirmTransfer/TransactionPopup.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+    AccountTransactionPayload,
+    AccountTransactionType,
+    isBakerAccount,
+    isDelegatorAccount,
+    ConfigureBakerPayload,
+    ConfigureDelegationPayload,
+} from '@concordium/web-sdk';
+import { useSelectedAccountInfo } from '@popup/shared/AccountInfoListenerContext/AccountInfoListenerContext';
+import { useBlockChainParametersV1 } from '@popup/shared/BlockChainParametersProvider';
+import Modal from '@popup/shared/Modal';
+import Button from '@popup/shared/Button';
+
+interface Props {
+    payload: AccountTransactionPayload;
+    transactionType: AccountTransactionType;
+    setShowPopup: (show: boolean) => void;
+    showPopup: boolean;
+}
+
+/**
+ * Component that displays a popup with a message related to a transaction when sending it.
+ * Note that only some transactions warrant a popup/message.
+ */
+export default function TransactionPopup({ transactionType, payload, showPopup, setShowPopup }: Props) {
+    const { t } = useTranslation('account', { keyPrefix: 'transactionPopup' });
+    const { t: tShared } = useTranslation('shared');
+    const accountInfo = useSelectedAccountInfo();
+    const parametersV1 = useBlockChainParametersV1();
+
+    const message = useMemo(() => {
+        if (accountInfo) {
+            switch (transactionType) {
+                case AccountTransactionType.ConfigureBaker: {
+                    if (isBakerAccount(accountInfo)) {
+                        const newStake = (payload as ConfigureBakerPayload).stake?.microCcdAmount;
+                        if (!newStake || accountInfo.accountBaker.stakedAmount < newStake) {
+                            if ((payload as ConfigureBakerPayload).keys) {
+                                return t('configureBaker.updateKeys');
+                            }
+                            return t('configureBaker.update');
+                        }
+                    } else {
+                        return t('configureBaker.start');
+                    }
+                    break;
+                }
+                case AccountTransactionType.ConfigureDelegation: {
+                    if (isDelegatorAccount(accountInfo)) {
+                        const newStake = (payload as ConfigureDelegationPayload).stake?.microCcdAmount;
+                        if (!newStake || accountInfo.accountDelegation.stakedAmount < newStake) {
```

Comment:
This is true when `newStake` is `0`, i.e. when I stop delegating. The same issue is there for baking.

Reply:
True